osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/PathControlPointPiece.cs

```diff
@@ -209,8 +193,8 @@ private void updateMarkerDisplay()
             if (IsHovered || IsSelected.Value)
                 colour = colour.Lighten(1);
 
-            marker.Colour = colour;
-            marker.Scale = new Vector2(hitObject.Scale);
+            Colour = colour;
+            Scale = new Vector2(hitObject.Scale);
```

Comment:
Generally the reason we add the nested container in the first place is to isolate things like colour/scale from acting on `this`. It avoids cases where a drawable changes these values, but then they are changed from outside and overwrite the internal values.

Because this is a very local usage I'm willing to let this slide but keep this in mind for more generic cases.